### PR TITLE
Translate Rails Doctrine into Traditional Chinese

### PR DESCRIPTION
**Pull request by JuanitoFatas**
Screenshot:

screenshot 2018-03-15 00 35 04

@bugtender please review the content 🙏

### CODE REVIEW

File: doctrine/zh_tw.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+      <p>但執行流程與關注的地方有著微妙的不同。第一個語句，關注的是群體是否有特定的人。第二個語句則是人是否屬於群體。兩條語句都差不多長，但我主張第二條語句更美，能讓我在想出這條件式考量的是人的時候，更容易有種會心一笑的感覺。</p>
+
+      <h2 id="provide-sharp-knives">提供實用工具</h2>
+
+      <p>Ruby 本身就有許多實用的工具，不是碰巧，設計正是如此。最出名的就是 Monkey Patching：可以修改類別與方法的能力。</p>
+
+      <p>但這個能力很快被嘲諷說，一般的程式設計師掌握不了。而從限制性較嚴格的語言陣營人們認為，Ruby 這個功能盲目的信任程式設計師，Ruby 是註定是要失敗的。</p>
+
+      <p>什麼都可以改變的話，又怎麼能阻止你把 <code>String#capitalize</code> 从原来处理 <code>"something bold".capitalize</code> 改成 <code>"Something Bold"</code> 而不是本來的 <code>"Something bold"</code> 呢？這在你自己的應用上可能沒問題，但這就把其它倚賴 <code>String#capitalize</code> 的函式庫都搞壞了。</p>
```
Comment (bugtender) [inline doctrine/zh_tw.html]: `从原来处理` should be `從原來處理`.

File: doctrine/zh_tw.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+      <p>總結，追求幸福可能是形成 Ruby on Rails 的關鍵要素。而這，會跟著 Rails 一直走下去。</p>
+
+      <h2 id="convention-over-configuration">約定優於配置</h2>
+
+      <p>早期 Rails 的生產力座右銘是：“你不是獨一無二的，大家都一樣”。只要放下了自負的個人喜好，便可以跳過無謂的世俗決定，而只在最重要的地方下更快的決定。</p>
+
+      <p>真的有人在乎資料庫主鍵的名稱嗎？用 <code>id</code>、<code>postID</code>、<code>posts_id</code> 或 <code>pid</code>，真的有差別嗎？這真的值得反覆討論嗎？不！</p>
+
+      <p>Rails 部分使命就是，幫開發者在為 Web 打造資訊系統，日漸複雜龐大的決策叢林中揮荊斬棘。有幾千個無謂的決策只需要做一次，而若是別人幫你做這些決定，那就更好了。</p>
```
Comment (bugtender) [inline doctrine/zh_tw.html]: I'm am confuse which one is better for here.  `揮荊斬棘`、`劈荊斬棘`、`披荊斬棘`.  ([zh-TW Dict](http://dict.revised.moe.edu.tw/cgi-bin/cbdic/gsweb.cgi?ccd=RfGxS_&o=e0&sec1=1&op=sid=%22Z00000024530%22.&v=-2))

**Comment by bugtender** [comment]
@JuanitoFatas Everything looks good.😃 Just a little `zh-TW` / `zh-CN` bug I mention.

File: doctrine/zh_tw.html (hunk 2 above)
Comment (JuanitoFatas) [reply doctrine/zh_tw.html]: `披荊斬棘` 👍

**Comment by JuanitoFatas** [comment]
Thanks! ❤️

**Pull request merged**